src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -128,4 +128,31 @@ partial class BoundThrowExpression
     {
         protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.Expression);
     }
+
+    internal partial class BoundMethodGroup
+    {
+        protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.ReceiverOpt);
```

Comment:
>protected override ImmutableArray<BoundNode> Children => ImmutableArray.Create<BoundNode>(this.ReceiverOpt); [](start = 8, length = 108)

Instead of duplicating this for BoundMethodGroup and BoundPropertyGroup, consider overriding this property once in BoundMethodOrPropertyGroup

Reply:
This isn't a blocker.

---
In reply to: [140047143](https://github.com/dotnet/roslyn/pull/21857#discussion_r140047143) [](ancestors = 140047143)